circus-train-core/src/main/java/com/hotels/bdp/circustrain/core/replica/hive/DropTableService.java

```diff
@@ -56,13 +100,52 @@ public void removeTableParamsAndDrop(
       }
       client.alter_table(databaseName, tableName, table);
     }
-    LOG
-        .info("Dropping table '{}.{}'.", table.getDbName(), table.getTableName());
-    client.dropTable(table.getDbName(), table.getTableName(), false, true);
+    LOG.info("Dropping table '{}.{}'.", databaseName, tableName);
+    client.dropTable(databaseName, tableName, false, true);
+  }
+
+  private void deleteData(DataManipulationClient dataManipulationClient, String replicaDataLocation) {
+    try {
+      LOG.info("Dropping table data from location: {}.", replicaDataLocation);
+      boolean dataDeleted = dataManipulationClient.delete(replicaDataLocation);
+      LOG.info("Data deleted: {}.", dataDeleted);
+    } catch (IOException e) {
+      LOG.info("Could not drop replica table data at location:{}.", replicaDataLocation);
+    }
+  }
+
+  private void deletePartitionData(
+      DataManipulationClient dataManipulationClient,
+      String replicaTableLocation,
+      List<String> replicaPartitionLocations) {
+    try {
+      LOG.info("Dropping partition data from base location: {}.", replicaTableLocation);
+      for (String location : replicaPartitionLocations) {
+        boolean deleted = dataManipulationClient.delete(location);
+        LOG.debug("Attempted to delete data from location: {}. Successful deletion = {}.", location, deleted);
+      }
+    } catch (IOException e) {
+      LOG.info("Could not drop replica partition data at location:{}.", replicaTableLocation);
```

Comment:
Then this message can log the actual partition location which will be more meaningful (and also log the exception)